lib/notifications/display.dart

```diff
@@ -37,11 +37,12 @@ enum NotificationSound {
 class NotificationChannelManager {
   /// The channel ID we use for our one notification channel, which we use for
   /// all notifications.
-  // TODO(launch) check this doesn't match zulip-mobile's current or previous
```

Comment:
And I just checked with a grep, and there are no other `TODO(launch)` comments in the tree :slightly_smiling_face:.

(And the only TODO(#715) comments are the ones resolved by #1577.)